js/views/mediacontrolsview.js

```diff
@@ -320,7 +320,7 @@
 
 						if (extensionURL) {
 							var text = t('spreed', 'Screensharing extension is required to share your screen.');
-							var element = $('<a>').attr('href', extensionURL).attr('target','_blank').text(text);
+							var element = '<a href="' + extensionURL + '" target="_blank">' + text + '</a>';
```

Comment:
In secured translations we trust....
Or we add a escapeHtml just to be on the save side?

Reply:
`escapeHTML` added.